pkg/api/validate/openshiftcluster_validatedynamic.go

```diff
@@ -36,57 +46,50 @@ func (*azureClaim) Valid() error {
 
 type openShiftClusterDynamicValidator struct {
 	env env.Interface
-
-	subnet subnet.Manager
```

Comment:
All changes below in this file are just to get rid of data race when using a single validator instance in `frontend` to validate multiple objects (previously we were creating an instance of `openShiftClusterDynamicValidator` for each request).

This does not change the logic: we just pass `subnet.Manager` and `*api.OpenShiftCluster` in the arguments istead using struct fields.

Reply:
This style of annotating PRs with notes is extremely useful, thank-you.